CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
   platform for the generated documentation.
   ([Jiangda Wang](https://github.com/frank-iii))
 
+- `gleam new` now takes an optional `--template` flag (with options `erlang`
+  and `javascript`) to specify the template and target for the generated
+  project.
```

Comment:
The flag already existed, it's that there are new values and "erlang" replaces "lib"